README.md

```diff
@@ -58,6 +58,58 @@ Response:
 OK
 ```
 
+### `GET /retrievable-deals/miner/:minerId`
+
+Parameters:
+- `minerId` - a miner id like `f0814049`
+
+Response:
+
+Number of deals grouped by client IDs.
+
+```json
+[
```

Comment:
We could wrap the response in 

```json
{ 
    "minerId": "<minerId>",
    "deal-stats": [
        ...
    ]
}
```

Reply:
If we adopt my proposal above (`GET /miner/{minerId}/retrievable-deals/summary`), then I propose the following response body format:

```json5
{
  "minerId": "<minerId>",
  "clients": [
    // ...
  ]
}
```